docs/spec.yml

```diff
@@ -304,6 +303,260 @@ paths:
           description: Unprocessable request
         '500':
           description: Validation Error
+  '/v3/technical-records/updateVrm/{systemNumber}/{createdTimestamp}':
+    patch:
+      summary: 'Update VRM on a technical record'
+      tags:
+        - Update VRM
+      parameters:
+        - in: path
+          name: systemNumber
+          schema:
+            type: string
+            minLength: 3
+            maxLength: 21
+          required: true
+          description: This represents the systemNumber of the vehicle. Partition key of the DynamoDB table.
+        - in: path
+          name: createdTimestamp
+          schema:
+            type: string
+            pattern: \d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{0,20}Z
```

Comment:
a valid timestamp is 2019-06-24T10:26:56.903Z. I don't think that, that will match the pattern here

Reply:
This is the pattern used in `validateSysNumTimestampPathParams`. Are we seeing issues with requests to this endpoint at the moment with invalid timestamps?